force-app/main/default/classes/tasks/BasicTasks.cls

```diff
@@ -24,7 +33,19 @@ public class BasicTasks {
   public static Decimal mathPower(Integer a, Integer n) {
     //write your code below this line
     Decimal result = 0;
-
+    Decimal c = 0;
+    Decimal d = 1;
+    if (n == 0) {
```

Comment:
Can you do it without this `if`?

Reply:
Hey, updated the file. set the results for 1 be default and the loop won't start unless n>0